src/randoop/operation/ConstructorCall.java

```diff
@@ -216,31 +220,68 @@ public ExecutionOutcome execute(Object[] statementInput, PrintStream out) {
   }
 
   /**
-   * Returns the return type of this constructor.
+   * {@inheritDoc}
+   * @return type of the object created (i.e., class for constructor).
    */
+  @Override
   public Class<?> getOutputType() {
-    if (outputTypeCached == null) {
-      outputTypeCached = constructor.getDeclaringClass();
-    }
     return outputTypeCached;
   }
 
   /**
-   * A string representing this constructor. The string is of the form:
-   *
-   * CONSTRUCTOR
+   * {@inheritDoc}
+   * Generates a string representation of the constructor signature.
    *
-   * Where CONSTRUCTOR is a string representation of the constrctor signature. Examples:
-   *
-   * java.util.ArrayList.&lt;init&gt;()
-   * java.util.ArrayList.&lt;init&gt;(java.util.Collection)
+   * Examples:
+   *  java.util.ArrayList.&lt;init&gt;()
```

Comment:
To put these examples on separate lines in the generated HTML, you should probably put them within <pre>...</pre>.